sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -10,11 +10,10 @@ skip-csproj: true
 model-namespace: false
 public-clients: false
 head-as-boolean: false
-mgmt-debug:
-  show-request-path: true
+
 batch:
-  - tag: package-common-type-2022-04
-  - tag: package-resources-2022-04
+  # - tag: package-common-type-2022-04
```

Comment:
local temporally change?

Reply:
I forgot to revert this... will update this soon

Reply:
I reverted this and there is no change in resourcemanager.